### PR TITLE
data: anurag-dhamala

### PR DESCRIPTION
**Pull request by anurag-dhamala**

### CODE REVIEW

File: data/anurag-dhamala.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+  "name": "Anurag Dhamala",
+  "type": "personal",
+  "bio": "Software Engineer who loves to dive deep into new technologies and collaborate with like-minded people",
+  "avatar": "https://avatars.githubusercontent.com/u/50538732?v=4",
```
Comment (SurajPratap10) [inline data/anurag-dhamala.json:5]:
```suggestion

```

Comment (SurajPratap10) [reply data/anurag-dhamala.json:5]: We don't need the `avatar` field now.

Comment (anurag-dhamala) [reply data/anurag-dhamala.json]: Removed the avatar field.

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

**Review by SurajPratap10 — APPROVED** [review]
LGTM 🎉

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**